extensions/resource-deployment/src/ui/notebookWizard/notebookWizard.ts

```diff
@@ -58,7 +59,9 @@ export class NotebookWizard extends WizardBase<NotebookWizard, NotebookWizardPag
 			const notebook = await this.prepareNotebookAndEnvironment();
 			await this.openNotebook(notebook);
 		} catch (error) {
-			vscode.window.showErrorMessage(error);
+			console.log(error);
```

Comment:
So what's the behavior now - I see above we're setting the container message with the error message but we're also showing the error message in a toast? Please include a screenshot of what this looks like when an error occurs

Reply:
Yes, we are showing both the combating message and the toast

Reply:
Well that seems a bit redundant. Any reason why you went that way instead of just showing one?

Reply:
Let me redo this to show just one.

Reply:
Updated description with the screenshot.